Formula/picat.rb

```diff
@@ -0,0 +1,23 @@
+class Picat < Formula
+  desc "Simple, and yet powerful, logic-based multi-paradigm programming language"
+  homepage "http://picat-lang.org"
+  url "http://picat-lang.org/download/picat24_src.tar.gz"
+  version "2.4.8"
+  sha256 "72b452a8ba94d6187d837dcdb46aab0d7dc724651bac99a8cf2ada5c0a3543dd"
+  depends_on "gcc" => :build
+
+  def install
+    inreplace "emu/Makefile.picat.mac64", "/usr/local/bin/gcc", HOMEBREW_PREFIX/"bin/gcc-8"
```

Comment:
I think `#{Formula["gcc"].opt_bin}/gcc` might be cleaner (if it works.) because this would break if gcc-8 ceases to exist.

Reply:
Why is gcc required?

Reply:
Thanks for the suggestion, basically it is the same. #{Formula["gcc"].opt_bin} contains gcc-8 only as well. I also wonder why there's no symlink to gcc.